www/css/cytube.css

```diff
@@ -239,8 +241,9 @@ li.ui-sortable-helper, li.ui-sortable-placeholder + li.queue_entry {
 .server-msg-disconnect {
     border: 1px solid #ff0000;
     padding-top: 5px;
-    padding-bottom: 5px;
-    margin: 5px;
+    margin-top: 5px;
+    margin-bottom: 5px;
+    height: 30px;
```

Comment:
I don't think that setting an explicit pixel height for the box is the right way to solve your problem of "correct" size.  Different themes (or channel CSS, or stylish, or whatnot) may use a different font size or line-height, and furthermore in the case of `.server-msg-disconnect`, if someone is on a small screen and receives a long error message, it will be linewrapped, hence breaking the explicit height.

Currently, the height is automatic with `padding-bottom` and `padding-top` being set to prevent the box from being too narrow around the text.  I'm confused why you removed `padding-bottom` and set `height` instead-- is there some reason that the automatic height calculation breaks specifically for Modern?